src/apis/fmiApi.ts

```diff
@@ -29,7 +29,7 @@ export async function GetRawXMLResponse(query: string) {
   return res
 }
 
-export async function GetSimpleFmiResponse(
+export default async function GetSimpleFmiResponse(
```

Comment:
Mä en saanu tämän funktion stubbausta toimimaan ilman että tein tästä defaultin, mutta ei siitä varmaan mitään haittaa ole?

Reply:
Joo ei ton pitäis kai mitään kummempaa aiheuttaa